src/view/selection.js

```diff
@@ -461,6 +462,25 @@ export default class Selection {
 	}
 
 	/**
+	 * Returns selected element. {@link engine.view.Element Element} is considered as selected if there is only
```

Comment:
Returns the selected element.

one range in the selection

and that range contains exactly one element.